src/python/impactx/dashboard/Toolbar/examples_loader/loader.py

```diff
@@ -0,0 +1,91 @@
+"""
+This file is part of ImpactX
+Copyright 2025 ImpactX contributors
+Authors: Parthib Roy
+License: BSD-3-Clause-LBNL
+"""
+
+from pathlib import Path
+
+from ... import state
+from ...Input.utils import GeneralFunctions
+from ..file_imports.ui_populator import populate_impactx_simulation_file_to_ui
+
+state.impactx_example_list = []
+
+
+DASHBOARD_EXAMPLES = {
+    "fodo/run_fodo.py",
+    "chicane/run_chicane_csr.py",
+    "fodo_space_charge/run_fodo_envelope_sc.py",
+    "apochromatic/run_apochromatic.py",
+    "kurth/run_kurth_10nC_periodic.py",
+    "expanding_beam/run_expanding_fft.py",
+    "expanding_beam/run_expanding_envelope.py",
+    # "iota_lattice/run_iotalattice.py",
+    "cyclotron/run_cyclotron.py",
+    "dogleg/run_dogleg.py",

```

Comment:
Let's comment them in again with the parser PR, currently only FODO works:
```suggestion
    "fodo/run_fodo.py",
    # "chicane/run_chicane_csr.py",
    # "fodo_space_charge/run_fodo_envelope_sc.py",
    # "apochromatic/run_apochromatic.py",
    # "kurth/run_kurth_10nC_periodic.py",
    # "expanding_beam/run_expanding_fft.py",
    # "expanding_beam/run_expanding_envelope.py",
    # "iota_lattice/run_iotalattice.py",
    # "cyclotron/run_cyclotron.py",
    # "dogleg/run_dogleg.py",

```